File: reloadModules.js
const fs = require('fs');
module.exports = {
  name:"reloadModules",
  shownInHelp:"false",
  description:"",
  helpMessage:"",
  permission:0,
  func : function (msg, args) {
    if ((msg === undefined) || (msg.member.hasPermission('ADMINISTRATOR'))) {
      if (msg !== undefined){var react = msg.react("✅");}
      fs.readdirSync("./modules/").forEach(file => {
        if (file.slice(-3) === ".js"){
          ModuleName = file.slice(0, -3);
          if (Modules[ModuleName] && ModuleName != "reloadModules") {
            delete require.cache[require.resolve("./modules/" + ModuleName)];
            delete Modules[ModuleName];
            console.log("The module '"+ ModuleName + "' has been unloaded");
          }
          try {
            Modules[ModuleName] = require("./modules/" + ModuleName);
            console.log("The module '"+ ModuleName + "' has beeen loaded");
          } catch (e) {
            if (msg !== undefined) {
              react.remove(client.user);
              msg.react("❎");
            }   
            console.log("The module '" + ModuleName + "' failed to be loaded");
            console.warn(e);
          }
        }
      });
    }
  }
}

File: reloadModules.py
from types import FunctionType
import os
import sys
class reloadModules:
    name = "reloadModules"
    category = "Misc"
    shownInHelp = False
    description = ""
    helpMessage = ""
    permission = 8

    async def function(self, message, args):
        if(message):
            await message.add_reaction("☑")
        sys.path.append('./modules')
        for module in os.listdir('./modules'):
            if module.endswith('.py'):
                fmodule = module[:-3]
                if (fmodule in self.Modules):
                    del self.Modules[fmodule]
                    print("The module '" + fmodule + "' has been unloaded.")
                try:
                    self.Modules[fmodule] = __import__(fmodule, fromlist=[fmodule])
                    print("The module '" + fmodule + "' has been loaded.")
                except Exception:
                    await message.remove_reaction("☑", self.client.user)
                    await message.add_reaction("❌")
                    print("The module" + fmodule + " failed to be loaded.")